client/src/components/body/lab4/reading.js

```diff
@@ -0,0 +1,86 @@
+import React, {useEffect} from "react";
+import {LAB_ID} from '../../../constants/lab4';
+import UserLabService from '../../../services/UserLabService';
+import {Pie} from "react-chartjs-2";
+
+const data = {
+    labels: [
+        "US Population (Millions)",
+        "Population with Dexterity Impairments (Millions)"
+    ],
+    datasets: [
+        {
+            label: "Dexterity Impaired in a Population of 328 Million People",
+            borderColor: "black",
+            backgroundColor: ["blue", "red"],
+            data: [328, 40.7],
+            borderWidth: "1"
+        }
+    ]
+};
+
+const Reading = () => {
+    useEffect(() => {
+        return () => {
+            UserLabService.complete_reading(LAB_ID);
+        }
+    });
+    return (
+        <div className="study">
+            <h3>What Is a Dexterity Impairment?</h3>
+            <p>
+                Many types of dexterity impairments exist, but all limit the functionality of one or more limbs and cause the loss of fine control of movement.
+                They can be caused by an injury, a genetic disorder, or a disease.
+                For example, Muscular dystrophy is a genetic disorder that causes progressive weakness in the muscles.
+            </p>
+
+            <h3>Examples of Dexterity Impairments</h3>
+            <p>
+                <ul className="study__list">
+                    <li>Spinal cord injury</li>
```

Comment:
Maybe this list could be organized differently - there's a ton of empty white space to the right of this list

Reply:
That's true, I think we'll be discussing how we want to reformat long lists